presto-hive/src/main/java/com/facebook/presto/hive/HiveWritableTableHandle.java

```diff
@@ -43,7 +44,8 @@ public HiveWritableTableHandle(
             LocationHandle locationHandle,
             Optional<HiveBucketProperty> bucketProperty,
             HiveStorageFormat tableStorageFormat,
-            HiveStorageFormat partitionStorageFormat)
+            HiveStorageFormat partitionStorageFormat,
+            HiveCompressionCodec compressionCodec)
```

Comment:
This new field is not used in commit "Add compression_codec session property in Hive connector" right?